### DISCUSSION TITLE
Welcome to frontend-html-css-js Discussions!

### DISCUSSION THREAD

**Discussion by ghost** [Dezenix/frontend-html-css-js#9] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Pratyush-IITBHU** [comment]
Respected Sir,

I am Pratyush, an electrical sophomore at IIT (BHU) Varanasi. I am very much interested in the 
frontend-html-css-js  project of Dezenix.

I have some knowledge of Python, Javascript , CSS , HTML, and design in Adobe illustrator. So I want to ask how to start and prepare for this project and what prerequisites I need for completing it.

Please kindly inform me about the topics the website is going to cover.

I also request you to please guide me in making my proposal, because I am eagerly interested in this project!

Thank You
